src/pixel/sender.js

```diff
@@ -13,25 +16,50 @@ export function PixelSender (liveConnectConfig, onload, presend) {
    * @param {StateWrapper} state
    * @private
    */
-  function _send (state) {
+  function _sendAjax (state) {
+    _sendState(state, 'j', uri => {
+      get(uri, responseBody => {
+        if (isFunction(onload)) onload()
+        const bakers = safeParseJson(responseBody).bakers
+        if (isArray(bakers)) {
+          for (let i = 0; i < bakers.length; i++) sendPixel(`${bakers[i]}?dtstmp=${utcMillis()}`)
+        }
+      })
+    })
+  }
+
+  /**
+   * @param {StateWrapper} state
+   * @private
+   */
+  function _sendPixel (state) {
+    _sendState(state, 'p', uri => {
+      sendPixel(uri, onload)
+    })
+  }
+
+  function _sendState (state, endpoint, makeCall) {
     if (state.sendsPixel()) {
```

Comment:
does this one mean that it sends only pixel? Or is it now `send in general`?

Reply:
"sendsPixel" now means "send via any medium".But I don't know how to rename this and I am fine keeping it as "send pixel".